games-puzzle/pipewalker/pipewalker-0.9.4-r2.ebuild

```diff
@@ -11,16 +11,22 @@ SRC_URI="mirror://sourceforge/pipewalker/${P}.tar.gz"
 LICENSE="GPL-3"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 
-DEPEND="media-libs/libsdl[opengl,video]
+DEPEND="
+	media-libs/libpng:=
+	media-libs/libsdl[opengl,sound,video]
 	virtual/opengl
-	virtual/glu"
+"
 RDEPEND="${DEPEND}"
 
+src_prepare() {
+	default
+	eautoreconf
+}
+
 src_configure() {
-	append-flags $(sdl-config --cflags)
-	econf --datadir="/usr/share"
+	append-cflags $(sdl-config --cflags)
```

Comment:
check exactly what `sdl-config --cflags` returns here, because my feeling is this might need to be `append-cppflags`

Reply:
If you're needing to get flags manually like this then I'd much prefer you use `pkg-config` (`tc-getPKG_CONFIG`) instead.

Reply:
econf is probably using sdl-config --libs /still/, but pkg-config is still more right even if it mismatches. Optimally I wish sdl-config wasn't used at all.

That aside, it's often overlooked but sdl-config/pkg-config calls should get `|| die` as well, I had a case where build succeeded despite pkg-config failed and resulted in an underlinked shared library causing issues for other packages.

Usually I do something [like this](https://github.com/gentoo/gentoo/blob/cf72416b29d1fc39f978044b7da63d909d90bd24/games-rpg/openglad/openglad-0.98_p20200907.ebuild#L51) anyhow.

Reply:
Done